frontend/src/App.js

```diff
@@ -1,5 +1,5 @@
-import logo from './logo.svg';
-import './App.css';
+import logo from "./logo.svg";
+import "./App.css";
 
 function App() {
   return (
```

Comment:
Why isn't the navbar hooked up here?

Reply:
good question, somehow my App.js file didnt get pushed